source/connect.html.erb

```diff
@@ -10,6 +10,64 @@ title: Help with Connecting to GovWifi
 
     <div class="column-two-thirds">
       <h1 class="heading-large">How to connect to GovWifi</h1>
+
+      <p class="grey-text-box">
+        By connecting to GovWifi, you accept the <%= link_to "terms and conditions", "https://www.gov.uk/government/publications/terms-and-conditions-for-connecting-to-govwifi/terms-and-conditions-for-connecting-to-govwifi"%> ,
+        and our <%= link_to "privacy notice", "https://www.gov.uk/government/publications/govwifi-privacy-notice" %>.
+      </p>
+
+      <p>
+        GovWifi is a secure wifi service, developed by Government Digital Service
+        (GDS). It’s currently in public beta, which means we’re still testing and
```

Comment:
Just to double-check - this is still true? We're still in beta?

Reply:
Yeah its true